lxd/storage/drivers/driver_btrfs_patches.go

```diff
@@ -0,0 +1,60 @@
+package drivers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/lxc/lxd/lxd/db"
+)
+
+func (d *btrfs) patchBucketNames() error {
+	var buckets map[string]*db.StorageBucket
+
+	err := d.state.DB.Cluster.Transaction(context.TODO(), func(ctx context.Context, tx *db.ClusterTx) error {
```

Comment:
All storage drivers have access to the daemon state via `d.state`. Instead of using `context.TODO()` we should use `d.state.ShutdownCtx`.